### PR TITLE
fabtests/unexpected_msg.c: Allow running the test with FI_DELIVERY_COMPLETE

### PR DESCRIPTION
**Pull request by shijin-aws**
Add the -U option like other tests do to allow transmission with FI_DELIVERY_COMPLETE

### CODE REVIEW

**Review by wenduwan — APPROVED** [review]

File: fabtests/functional/unexpected_msg.c [hunk 1]
```diff
@@ -369,6 +372,7 @@ int main(int argc, char **argv)
 			FT_PRINT_OPTS_USAGE("-v", "Enable data verification");
 			FT_PRINT_OPTS_USAGE("-C", "transfer remote CQ data");
 			FT_PRINT_OPTS_USAGE("-M <count>", "number of concurrent msgs");
+			FT_PRINT_OPTS_USAGE("-U", "Do transmission with FI_DELIVER_COMPLETE");
```
Comment (aingerson) [inline fabtests/functional/unexpected_msg.c:375]: misspelled DELIVERY

Comment (shijin-aws) [reply fabtests/functional/unexpected_msg.c]: Thanks! fixed

**Comment by shijin-aws** [comment]
I think `-U` should be included in CS_OPTS so we don't have to add it in each tests separately. But I will make that in a separate PR

**Review by aingerson — APPROVED** [review]

**Comment by shijin-aws** [comment]
I have a few follow-up patches on fabtests based on this. They are all nice-to-have improvements for main and 1.19.x, but not critical for 1.19 release. Let me know if I need to pause backporting to 1.19.x branch @shefty

**Pull request merged**